colibre/description.html

```diff
@@ -159,29 +187,60 @@ <h4>AGN</h4>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:with_krumholz_vorticity", 0) | int }}</td>
     </tr>
     <tr>
         <td>Use Angular Momentum Limiter</td>
         <td>{{ data.metadata.parameters.get("COLIBREAGN:with_angmom_limiter", 0) | int }}</td>
     </tr>
     <tr>
-        <td>Reposition Velocity</td>
-        <td>{{ data.metadata.parameters.get("COLIBREAGN:reposition_coefficient_upsilon", 0.0) | float | format_number }}</td>
+        <td>Normalisation of the viscous time-scale</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:viscous_alpha", 0.0) | float | format_number }}</td>
     </tr>
     <tr>
-        <td>Max Reposition Mass</td>
-        <td>{{ data.metadata.parameters.get("COLIBREAGN:reposition_coefficient_upsilon", 0.0) | float | format_number }} M\(_\odot\)</td>
+        <td>Use Nibbling</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:use_nibbling", 0) | int }}</td>
     </tr>
     <tr>
-        <td>With Reposition Threshold</td>
+        <td>Minimal Gas Mass For Nibbling</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:min_gas_mass_for_nibbling", 0.0) | float | format_number }} M\(_\odot\)</td>
+    </tr>
+    <tr>
+        <td>Maximal Reposition Mass</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:max_reposition_mass", 0.0) | float | format_number }} M\(_\odot\)</td>
+    </tr>
+    <tr>
+        <td>Use Reposition Velocity Threshold</td>
         <td>{{ data.metadata.parameters.get("COLIBREAGN:with_reposition_velocity_threshold", 0) | int }}</td>
     </tr>
     <tr>
-        <td>Maximal Velocity Offset</td>
-        <td>{{ data.metadata.parameters.get("COLIBREAGN:reposition_coefficient_upsilon", 0.0) | float | format_number }} \(c_{s, {\rm BH}}\)</td>
+        <td>Maximal Reposition Velocity Offset</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:max_reposition_velocity_ratio", 0.0) | float | format_number }} \(c_{s, {\rm BH}}\)</td>
+    </tr>
+    <tr>
+        <td>Minimal Reposition Velocity Threshold</td>
+	<td>{{ data.metadata.parameters.get("COLIBREAGN:min_reposition_velocity_threshold", 0.0) | float | format_number }} km s\(^{-1}\)</td>
+    </tr>
+    <tr>
+        <td>Set Black Hole Reposition Speed</td>
+	<td>{{ data.metadata.parameters.get("COLIBREAGN:set_reposition_speed", 0) | int }}</td>
+    </tr>
+    <tr>
+        <td>Repositioning Speed Normalisation</td>
+	<td>{{ data.metadata.parameters.get("COLIBREAGN:reposition_coefficient_upsilon", 0.0) | float | format_number }} km s\(^{-1}\) M\(_\odot\)\(^{-1}\)</td>
+    </tr>
+    <tr>
+        <td>Black Hole Merger Threshold Type</td>
+        <td>{{ data.metadata.parameters.get("COLIBREAGN:merger_threshold_type", 0) | int }}</td>
+    </tr>
+    <tr>
+        <td>Black Hole Merger Max Distance Ratio</td>
+        <td>{{ "%.4g" % (data.metadata.parameters["COLIBREAGN:merger_max_distance_ratio"] | float) }}</td>
```

Comment:
Use `get`. Also a lot of these names are quite long - do they appear in the table ok?

Reply:
![Screenshot from 2020-10-07 17-28-52](https://user-images.githubusercontent.com/20153933/95352724-ca8f5200-08c2-11eb-9849-8987627c2dc5.png)

The table looks fine. Well, okay, maybe there is a bit too much text. What do you think?